src/blas/backends/mklgpu/mkl_internal_blas_gpu_wrappers.cpp

```diff
@@ -24,1143 +24,1166 @@
 #include "mkl_internal_blas_gpu_wrappers.hpp"
 #include "mkl_internal_blas_sycl_gpu.hpp"
 
-namespace onemkl {
+namespace oneapi {
+namespace mkl {
 namespace mklgpu {
 namespace internal {
 
 // Buffer APIs
 
-void gemm(cl::sycl::queue &queue, onemkl::transpose transa, onemkl::transpose transb,
+void gemm(cl::sycl::queue &queue, oneapi::mkl::transpose transa, oneapi::mkl::transpose transb,
           std::int64_t m, std::int64_t n, std::int64_t k, float alpha,
           cl::sycl::buffer<float, 1> &a, std::int64_t lda, cl::sycl::buffer<float, 1> &b,
           std::int64_t ldb, float beta, cl::sycl::buffer<float, 1> &c, std::int64_t ldc) {
-    mkl::gpu::sgemm(queue, mkl::cblas_convert(transa), mkl::cblas_convert(transb), m, n, k, alpha,
-                    a, lda, b, ldb, beta, c, ldc);
+    ::mkl::gpu::sgemm(queue, ::mkl::cblas_convert(transa), ::mkl::cblas_convert(transb), m, n, k,
```

Comment:
The namespaces will be updated when beta09 is available, right? We will also change open source to use public APIs at some point.

Reply:
Another change to namespaces will come soon.